src/AgGrid.elm

```diff
@@ -358,6 +360,8 @@ storage. Column states can be restored through the `columnState` on the `GridCon
 -}
 type alias ColumnState =
     { aggFunc : Maybe String
+    , allowedAggFuncs : List String
+    , defaultAggFunc : Maybe String
```

Comment:
AgGrid does not specify `allowedAggFuncs` or `defaultAggFunc` in the ColumnState: https://www.ag-grid.com/javascript-data-grid/column-state/

So, we should not handle this either. Otherwise, we would just decode a hardcoded value basically, which does not make sense.

Reply:
Done (no idea why I thought we needed that)